eval/Containerfile

```diff
@@ -6,7 +6,6 @@ RUN dnf install -y python3.11 python3.11-devel git python3-pip make automake gcc
 
 WORKDIR /app
```

Comment:
Can you please also address the:

```suggestion
WORKDIR /app
```

and later

```
ENV HOME=/app

# TODO: this
RUN chmod -R 0777 /app
```

Why are you getting permission issues and why does it have to be `/app` folder?

Would something like `HF_HOME=/tmp` save you all the headaches? Or is the issue somewhere else?

Reply:
I was getting permission errors & wanted to quickly get past them, then never went back to it, lemme test to make sure all is good with /tmp - and might want to update this, too: https://github.com/redhat-et/ilab-on-ocp/blob/main/training/Containerfile#L14

Reply:
updated to /tmp all set